stbt.sh.in

```diff
@@ -44,7 +44,7 @@ case "$1" in
         usage; exit 0;;
     -v|--version)
         echo "stb-tester $STBT_VERSION"; exit 0;;
-    run|record|batch|config|control|lint|power|screenshot|templatematch|tv)
+    run|record|batch|config|control|docker|lint|power|screenshot|templatematch|tv)
```

Comment:
Would we want users to say `stbt docker run ...` or `stbt --docker run`? Seeing it's supposed to behave just like `stbt`...

On the other hand, `stbt docker --help` does make more sense than `stbt --docker --help`.